src/components/MetadataEditor/MetadataEditor.vue

```diff
@@ -24,9 +24,14 @@
       <div id="md-section-list">
         <form-section v-bind="sectionBinds('Sample_Information')" v-on="sectionEvents('Sample_Information')"/>
         <form-section v-bind="sectionBinds('Sample_Preparation')" v-on="sectionEvents('Sample_Preparation')"/>
-        <form-section v-bind="sectionBinds('Submitted_By')" v-on="sectionEvents('Submitted_By')"/>
         <form-section v-bind="sectionBinds('MS_Analysis')" v-on="sectionEvents('MS_Analysis')"/>
-        <form-section v-bind="sectionBinds('metaspace_options')" v-on="sectionEvents('metaspace_options')"/>
+        <form-section v-bind="sectionBinds('Submitted_By')" v-on="sectionEvents('Submitted_By')"/>
```

Comment:
I put `Submitted_By` in the wrong order in a previous refactor. After this change they're the same as prod again.